resources/qml/ObjectsList.qml

```diff
@@ -0,0 +1,267 @@
+// Copyright (c) 2017 Ultimaker B.V.
+// Cura is released under the terms of the LGPLv3 or higher.
+
+import QtQuick 2.2
+import QtQuick.Controls 1.1
```

Comment:
QtQuick Controls 2.0 is much faster, I think we should use those when writing new code (see refactor for sidebar in other PR).